lib/instructors.ts

```diff
@@ -1,36 +1,31 @@
-import fs from 'fs'
-import {compose, curry, map, pick, find, uniqBy} from 'lodash/fp'
+import {request} from 'graphql-request'
 
-const mapProps = compose(map, pick)
-const findPick = curry((selector, props, data) =>
-  pick(props, find(selector, data)),
-)
+const endpoint = 'https://egghead.io/graphql'
 
-const instructorsPath = './data/instructors.json'
-
-function readInstructors() {
-  const rawdata = fs.readFileSync(instructorsPath)
-  return uniqBy('slug', JSON.parse(rawdata.toString()))
-}
-
-export function getInstructors(props) {
-  const instructors = readInstructors()
+export async function loadInstructors(page: number = 1) {
+  const query = `query getInstructors($page: Int!){
+    instructors(per_page: 24, page:$page){
```

Comment:
I don't think there's a big need for pagination here. 

Might be wrong but I would generally prefer to not paginate when possible and I think we could load them all and aggressively cache. 

I actually think that instructors, like tags, make sense static. They don't change much and there is t a lot of them

Reply:
We briefly discussed "sorting" and "featuring" instructors based on the query and making /instructors more engaging in general. I think it's worth keeping dynamic to we can play with it more until we settle on features.